server/modules/user/api/userApi.js

```diff
@@ -149,10 +151,16 @@ export const init = app => {
   })
 
   const _updateUser = async (req, res) => {
-    const userToUpdate = Request.getBody(req)
-
+    const body = Request.getBody(req)
+    const { user: userToUpdateString } = body
+    let userToUpdate = A.parse(userToUpdateString)
     const validation = await UserValidator.validateUser(userToUpdate)
 
+    userToUpdate = {
+      ...userToUpdate,
+      props: A.stringify(userToUpdate.props),
+    }
```

Comment:
this is unclear. why that conversion?

Reply:
same as above, I felt dirty when I did that 👎